css/common/main.css

```diff
@@ -25,9 +25,14 @@ img {
   vertical-align: middle;
 }
 
+.container {
+  padding: 0 15px;
+  margin: 0 auto;
+}
+
 @media (min-width: 1200px) {
   .container {
-    max-width: 100%;
+    max-width: 1140px;
```

Comment:
What is I have an ultra wide monitor? This container will take 1140px of it and rest of it will be just blank?